starlette/status.py

```diff
@@ -72,6 +72,11 @@
 HTTP_510_NOT_EXTENDED = 510
 HTTP_511_NETWORK_AUTHENTICATION_REQUIRED = 511
 
+# for backwards compatibility
+HTTP_413_REQUEST_ENTITY_TOO_LARGE = HTTP_413_CONTENT_TOO_LARGE
+HTTP_414_REQUEST_URI_TOO_LONG = HTTP_414_URI_TOO_LONG
+HTTP_416_REQUESTED_RANGE_NOT_SATISFIABLE = HTTP_416_RANGE_NOT_SATISFIABLE
+HTTP_422_UNPROCESSABLE_ENTITY = HTTP_422_UNPROCESSABLE_CONTENT
```

Comment:
We need to use a `__getattr__` on the module to check those. Please check the history on this file, you'll find how to implement it.